modules/helm-filtering-helm-charts-by-certification-level.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="filtering-helm-charts-by-certification-level_{context}"]
+= Filtering Helm Charts by their certification level
+
+You can filter Helm charts based on their certification level in the *Developer Catalog*.
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and select a project.
+
+. From the *Developer Catalog* tile, select the *Helm Chart* option to see all the Helm charts in the *Developer Catalog*.
+
+. Use the filters to the left of the list of Helm charts to filter the required charts:
+* Use the *Chart Repositories* filter to filter charts provided by *Red Hat Certification Charts* or *OpenShift Helm Charts*.
+* Use the *Source* filter to filter charts sourced from *Partners*, *Community* or *Red Hat*. Certified charts are indicated with the icon.
+
+.Certified chart icon
```

Comment:
If we add the certified icon, then we can skip this screenshot.

Reply:
Done!